packages/framework/src/Form/Admin/Order/OrderPaymentFormType.php

```diff
@@ -27,14 +28,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'error_bubbling' => true,
             ])
             ->add('priceWithVat', MoneyType::class, [
-                'currency' => false,
+                'scale' => 6,
```

Comment:
I think the constraint below should be `NotNegativeMoneyAmount` the same is valid for other `OrderXXXFormType`

Reply:
I don't want to change the behavior. This is only an admin form type and OrderItems may have negative prices (eg. promo codes, rounding items, ...).